src/core/Akka/IO/Buffers/DirectBufferPool.cs

```diff
@@ -33,6 +33,22 @@ protected BufferPoolAllocationException(SerializationInfo info, StreamingContext
         }
     }
 
+    public class BufferPoolInfo
```

Comment:
Diagnostic class to get the state of the buffer pool without exposing the byte buffer class

Reply:
Is this still necessary if we're disabling the pool for both TCP and UDP?

Reply:
Ah nevermind, I got it. We still pool bytes but not SAEA.